doc/tutorial/mpi.rst

```diff
@@ -64,8 +64,8 @@ cells over 5 nodes, simulating 100 cells each.
 
 .. code-block::
 
-   srun -n NRANKS python network_ring_mpi.py
-   mpirun -n NRANKS python network_ring_mpi.py
+   srun -n NRANKS python mpi.py
```

Comment:
Should this be `${NRANKS}`, to make it bashy, and cut and paste freindly?

Reply:
I'm not expecting that variable is set, so I'd say it's good that the line will fail if left unedited. The context is quite clear about this, I think.

Reply:
Source code examples should be valid code. You could just say `mpirun -n 5 python mpi.py`.

Reply:
done